x-pack/plugins/spaces/public/plugin.tsx

```diff
@@ -67,33 +69,23 @@ export class SpacesPlugin implements Plugin<SpacesPluginSetup, SpacesPluginStart
       });
     }
 
+    if (plugins.savedObjectsManagement) {
+      const copySavedObjectsToSpaceService = new CopySavedObjectsToSpaceService();
+      copySavedObjectsToSpaceService.setup({
+        spacesManager: this.spacesManager,
+        notificationsSetup: core.notifications,
+        savedObjectsManagementSetup: plugins.savedObjectsManagement,
+      });
+    }
+
     spaceSelectorApp.create({
       getStartServices: core.getStartServices,
       application: core.application,
       spacesManager: this.spacesManager,
     });
 
     return {
-      registerLegacyAPI: (legacyAPI: LegacyAPI) => {
-        const copySavedObjectsToSpaceService = new CopySavedObjectsToSpaceService();
-        copySavedObjectsToSpaceService.setup({
-          spacesManager: this.spacesManager,
-          managementSetup: {
-            savedObjects: {
-              registry: {
-                register: action => legacyAPI.registerSavedObjectsManagementAction(action),
-                has: () => {
-                  throw new Error('not available in legacy shim');
-                },
-                get: () => {
-                  throw new Error('not available in legacy shim');
-                },
-              },
-            },
-          },
-          notificationsSetup: core.notifications,
-        });
-      },
+      registerLegacyAPI: (legacyAPI: LegacyAPI) => {},
```

Comment:
@elastic/kibana-security As I said on slack, this is now an empty function. Should I remove it and associated legacy call, or do you want to keep it in case of?

Reply:
Yes, if you're able to easily remove this, then please do so. If it becomes too much, I'll happily to it in a followup PR

Reply:
Done in https://github.com/elastic/kibana/pull/60481/commits/4b9931161b72079fde1b1947062562182b4ae8d9